src/themes/default/components/core/blocks/Header/MinimalHeader.vue

```diff
@@ -9,7 +9,9 @@
           <div class="col-sm-4 col-xs-2 middle-xs">
             <div>
               <template>
-                <return-icon class="p15 icon bg-cl-secondary pointer" to="/" />
+                <a class="inline-flex brdr-none p15 icon bg-cl-secondary pointer" href="/" data-testid="returnButton">
```

Comment:
```suggestion
                <router-link class="inline-flex brdr-none p15 icon bg-cl-secondary pointer" to="/" data-testid="returnButton">
```

Reply:
We need full page reload. We shouldn't use the router.